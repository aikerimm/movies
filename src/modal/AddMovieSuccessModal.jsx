import PropTypes from 'prop-types';
import './addMovieSuccessModal.css';

const AddMovieSuccessModal = ({ onClose }) => {
  return (
    <div className='overlay successOverlay' onClick={onClose}>
      <div
        className='successContainer'
        onClick={(event) => event.stopPropagation()}
      >
        <img src='/success.svg' className='successIcon' alt='success' />
        <p className='closeBtn' onClick={onClose}>
          X
        </p>
        <p className='titleText'>Congratulations!</p>
        <p className='promptText'>
          The movie has been added to database successfully
        </p>
      </div>
    </div>
  );
};

AddMovieSuccessModal.propTypes = {
  onClose: PropTypes.func.isRequired,
};

export default AddMovieSuccessModal;
