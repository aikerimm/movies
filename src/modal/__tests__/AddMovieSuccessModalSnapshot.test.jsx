import renderer from 'react-test-renderer';
import AddMovieSuccessModal from '../AddMovieSuccessModal';

it('AddMovieSuccessModal should render correctly', () => {
  const onClose = jest.fn();
  const tree = renderer.create(<AddMovieSuccessModal onClose={onClose}/>).toJSON();
  expect(tree).toMatchSnapshot();
});