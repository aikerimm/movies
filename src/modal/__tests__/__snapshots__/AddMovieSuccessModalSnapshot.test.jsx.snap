// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddMovieSuccessModal should render correctly 1`] = `
<div
  className="overlay successOverlay"
  onClick={[MockFunction]}
>
  <div
    className="successContainer"
    onClick={[Function]}
  >
    <img
      alt="success"
      className="successIcon"
      src="/success.svg"
    />
    <p
      className="closeBtn"
      onClick={[MockFunction]}
    >
      X
    </p>
    <p
      className="titleText"
    >
      Congratulations!
    </p>
    <p
      className="promptText"
    >
      The movie has been added to database successfully
    </p>
  </div>
</div>
`;
